templates/configurations/form.interface.hbs

```diff
@@ -66,6 +66,15 @@
       </div>
     </div>
 
+    <div v-if='_componentsWithSelectAll.includes(_componentName)' class="col-xs-12 form-group">
+      <div class="checkbox checkbox-icon">
+        <input type="checkbox" id="selectAll" name="selectAll" v-model="selectAll" v-on:input="required = false">
```

Comment:
Comment not addressed https://github.com/Fliplet/fliplet-widget-form-builder/pull/451/files#r923305925